app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -185,23 +191,17 @@ class RecentlyPlayedFragmentTest {
       profileId,
       timestampOlderThanOneWeek = true
     )
-    ActivityScenario.launch<RecentlyPlayedActivity>(
```

Comment:
aren't we removing other such launches in this class?
looks like the activity launches two times.

Reply:
Wouldn't that be wrong? I'd expect Espresso/Robolectric tests to only ever verify a single activity.

Reply:
We should update all tests. But this was just for example for one test.